static/lang/en.json

```diff
@@ -2201,9 +2201,16 @@
                 },
                 "ShowTokenIcon": "Show token icon?",
                 "Unidentified": "Unidentified?",
-                "AddBadge": "Add Counter",
+                "AddBadge": "Add",
+                "AddLabel": "Add Label",
+                "LabelPlaceholder": "Stage Label",
                 "BadgeType": {
-                    "Counter": "Counter"
+                    "counter": "Counter",
+                    "formula": "Formula",
+                    "value": "Value"
+                },
+                "ExtraSelections": {
```

Comment:
This fundamentally bothers me and I'm willing to try a replacement.

Reply:
"Fixed value" communicates it a little better

Reply:
For Mild/Moderate/Severe or is this about Value? I could buy it for Value.

My umbrage was at having an ExtraSelections section 🙃

Reply:
yeah what the fuck is that, and why is it bactrian'd

Reply:
Yes for value